benchmarks/dungeon-benchmark/src/main/java/com/squareup/sample/dungeon/benchmark/WorkflowBaselineProfiles.kt

```diff
@@ -0,0 +1,79 @@
+package com.squareup.sample.dungeon.benchmark
+
+import android.content.Context
+import androidx.benchmark.macro.CompilationMode
+import androidx.benchmark.macro.ExperimentalBaselineProfilesApi
+import androidx.benchmark.macro.junit4.BaselineProfileRule
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.BySelector
+import androidx.test.uiautomator.UiCollection
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObject
+import androidx.test.uiautomator.UiScrollable
+import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.Until
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.time.ExperimentalTime
+
+/**
+ * Shamelessly copied from the example the Google Benchmark team made on the Lottie library
+ * https://github.com/airbnb/lottie-android/pull/2005.
+ *
+ * This test actually generates the profiles.
+ */
+@RunWith(AndroidJUnit4::class)
+@OptIn(ExperimentalBaselineProfilesApi::class)
+public class WorkflowBaselineProfiles {
```

Comment:
Isn't this mostly the same code as the other test? Can we reuse some bits?

Reply:
Yes we can. :)